rudder-agent/SOURCES/uuidgen

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # uuidgen doesn't exist on AIX, so provide a compatible shell function
+# This is not compatible with Linux systems.
```

Comment:
Why not also add here a `if type uuidgen > /dev/null 2> /dev/null` protection, just in case, as we discussed?